DEV-GUIDE.md

```diff
@@ -195,6 +196,41 @@ undesirable as Golang may not erase out of date artifacts from the
 cache. In such a situation, erase the contents of `$GOPATH/pkg/` manually
 before rebuilding the code.
 
+### Building the code with debugger information
+
+If you'd like to use a debugger such as delve to debug istio containers, run
```

Comment:
what is delve? can you add a link to it?
Also, Istio (capitol I)

Reply:
+1

Reply:
Link to delve - and maybe some page on how to debug with it. The docker images for pilot/mixer/etc are based on scratch - I don't know how you could use delve with them.

I'm also +1 to base the debug images on xenial, and include delve in them - with only release based on scratch. 
We had this discussion before, it's a bit controversial.

Reply:
Sure, will add a link to delve and its tutorial

I've tried delve on pilot-discovery container that is built with scratch, and it works perfectly. But Squash doesn't work. More details in the below text.